stream_alert/alert_processor/outputs/pagerduty.py

```diff
@@ -401,6 +437,14 @@ def dispatch(self, **kwargs):
         if rule_context:
             rule_context = rule_context.get(self.__service__, {})
 
+        # Use the priority provided in the context, use it or the incident will be low priority
+        incident_priority = {}
+        _priority = rule_context.get('incident_priority', False)
+        if _priority:
+            verified_priority = self._priority_verify(_priority)
```

Comment:
Suggestion: pass the rule_context to the `_priority_verify` function and have it do all of this, and instead of having `_priority_verify` return `False` have it return the default value of `{}` or `dict()`..

So your call would just look something like:
`incident_priority = self._priority_verify(rule_context)`

And everywhere above that returns false would instead return `dict()`